extras/dml/data_mover_dml.c

```diff
@@ -128,6 +136,19 @@ data_mover_dml_operation_delete(void *data,
 	struct vdm_operation_output *output)
 {
 	dml_job_t *job = (dml_job_t *)data;
+	dml_status_t status = dml_check_job(job);
+	switch (status) {
+		case DML_STATUS_BEING_PROCESSED:
+			ASSERT(0 && "dml job being deleted during processing");
```

Comment:
Shouldn't there be a `break`? Or does this state also mean that the job is corrupted?

Reply:
Okay, I understand. Thank you for clarifying.